docs/IBCF.md

```diff
@@ -37,11 +37,18 @@ or by defining a default mapping to the trunk and exceptions for number ranges y
     *.3.2.1.2.4.2 IN NAPTR 1 1 "u" "E2U+sip" "!(^.*$)!sip:\\1@<local domain>!" .
     *.3.2.1.2.4.2.1 IN NAPTR 1 1 "u" "E2U+sip" "!(^.*$)!sip:\\1@<local domain>!" .
 
+You can also use ENUM to provide number portability information, for example
+
+    *.3.2.1.2.4.2 IN NAPTR 1 1 "u" "E2U+pstn:tel" "!(^.*$)!sip:\\1;npdi;rn=+242123@<local domain>!" .
+    *.3.2.1.2.4.2.1 IN NAPTR 1 1 "u" "E2U+pstn:tel" "!(^.*$)!sip:\\1;npdi@<local domain>!" .
+
 Refer to the [ENUM guide](ENUM) for more about how to configure ENUM.
 
 ## BGCF Configuration
```

Comment:
Do we talk about the default * route anywhere? If so, we should say it doesn't apply to routing numbers. (If not, we should.)